build/azure-devops/variables/build.yml

```diff
@@ -1,3 +1,3 @@
 variables:
-  DotNet.Sdk.Version: '8.0.201'
+  DotNet.Sdk.Version: '8.0.428'
```

Comment:
Looks like 8.0.412 is the most recent based on https://dotnet.microsoft.com/en-us/download/dotnet/8.0